aperturedb/queryMessage_pb2.py

```diff
@@ -3,81 +3,115 @@
 # source: queryMessage.proto
 
 import sys
```

Comment:
this file is automatically generated. We should not modify it.

Reply:
agreed. will revert